### PR TITLE
Check if the non-zero balance of the account when transfer

### PR DESCRIPTION
**Pull request by sirius651**
**Pull Request Summary**

### CODE REVIEW

File: src/hooks/xcm/useXcmBridge.ts [hunk 1]
```diff
@@ -357,6 +357,19 @@ export function useXcmBridge(selectedToken?: Ref<ChainAsset>) {
         return;
       }
 
+      // check if recipient account has non-zero native asset. (it cannot be transferred to an account with 0 nonce)
+      if (isValidEvmAddress(toAddress)) {
+        const balWei = await getBalance($web3.value!, toAddress);
+        if (Number(ethers.utils.formatEther(balWei)) === 0) {
+          throw Error('the balance of recipient account should be above zero');
```
Comment (bobo-k2) [inline src/hooks/xcm/useXcmBridge.ts:364]: Please put this into i18n file since the text is repeating

**Review by bobo-k2 — CHANGES_REQUESTED** [review]

**Comment by Kahonnohak** [comment]
Checked and it failed as it should be thanks!

Just to note, XCM has different UI - when Portal knows the receiver has less than required existential native token deposit, the transfer button is disabled. Here instead the button works and immediately fails. Fine for now. important that it fails.

**Review by impelcrypto — APPROVED** [review]

File: src/hooks/xcm/useXcmBridge.ts (hunk 1 above)
Comment (sirius651) [reply src/hooks/xcm/useXcmBridge.ts]: no idea how to use i18n in .ts file other than .vue. maybe in later, can tackle this. so refactored as separate variable for repeating words.

Comment (bobo-k2) [reply src/hooks/xcm/useXcmBridge.ts]: you can do something like.
```
const { t } = useI18n();

customMsg.value = t('dappStaking.toast.staked', {
            amount: stakeAmount,
            dapp: props.dapp.name,
});
```

Check StakePanel.vue for full implementation

Comment (sirius651) [reply src/hooks/xcm/useXcmBridge.ts]: Fixed

**Review by bobo-k2 — APPROVED** [review]
LGTM

**Pull request merged**